README.md

```diff
@@ -2,13 +2,25 @@
 <div align="center">
 
 # My Study Buddy
-[![Report Issue on Jira](https://img.shields.io/badge/Report%20Issues-Jira-0052CC?style=flat&logo=jira-software)](https://temple-cis-projects-in-cs.atlassian.net/jira/software/c/projects/DT/issues)
-[![Deploy Docs](https://github.com/ApplebaumIan/tu-cis-4398-docs-template/actions/workflows/deploy.yml/badge.svg)](https://github.com/ApplebaumIan/tu-cis-4398-docs-template/actions/workflows/deploy.yml)
-[![Documentation Website Link](https://img.shields.io/badge/-Documentation%20Website-brightgreen)](https://applebaumian.github.io/tu-cis-4398-docs-template/)
+[![Documentation Website Link](https://img.shields.io/badge/-Documentation%20Site-hotpink)](https://capstone-projects-2023-spring.github.io/project-virtual-pet/)
+[![Resources Link](https://img.shields.io/badge/-Resources-9cf)](https://capstone-projects-2023-spring.github.io/project-virtual-pet/resources/Links)
+
+[![Website](https://img.shields.io/website?down_color=red&style=plastic&url=http%3A%2F%2F68.183.30.203%3A3000%2F)](http://68.183.30.203:3000/)
 
 
 </div>
 
+## Directions
+
+* Please navigate to our app by clicking <a href="http://68.183.30.203:3000/">this link</a>. Once there, you can create an account, select a pet, and explore the site. 
```

Comment:
```suggestion
* Please navigate to our app by clicking [this link](http://68.183.30.203:3000/) Once there, you can create an account, select a pet, and explore the site. 
```
This may prevent odd formatting issues that I've noticed when mixing HTML and Markdown.